source/css/sass/patterns/templates/content-grid.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../functions/grid";
+@import "../../mixins/grid";
+@import "../../mixins/media-query";
+@import "../../variables/grid";
+
+.content-grid {
+  --primary-column-width: #{column-width($grid-main_column_count)};
+
+  @include base-grid($grid-max_width);
+  grid-template-areas:
+    ". extra-navigation ."
```

Comment:
`navigation`, or even `menu` is probably fine here. It's not the overall menu on the page, but this template doesn't know that.

Reply:
(Ideally there'd be a name that doesn't refer to what's in there, but for now...)